### PR TITLE
{chem}[gomkl/2021a] pyWannier90 v2021-12-07, PySCF v1.7.6, h5py v3.2.1, libcint v4.4.0 w/ Python 3.9.5

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)
requires:
- #16448

### CODE REVIEW

**Comment by boegel** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/p/pyWannier90/pyWannier90-2021-12-07-gomkl-2021a.eb [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+builddependencies = [
+    ('pybind11', '2.6.2'),
+]
+
+local_pymaj = '3'
+local_pymin = '9'
+local_pymajmin = local_pymaj + local_pymin
+
+dependencies = [
+    ('Python', '%s.%s.5' % (local_pymaj, local_pymin)),
```
Comment (jfgrimm) [inline easybuild/easyconfigs/p/pyWannier90/pyWannier90-2021-12-07-gomkl-2021a.eb:40]: as in #16447, the `local_py{maj,min,majmin}` variables seem redundant
```suggestion
dependencies = [
    ('Python', '3.9.5'),
```

**Review by jfgrimm — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/p/pyWannier90/pyWannier90-2021-12-07-gomkl-2021a.eb (hunk 1 above)
Comment (boegel) [reply easybuild/easyconfigs/p/pyWannier90/pyWannier90-2021-12-07-gomkl-2021a.eb]: fixed in 2f5588a, I think these are leftovers from an earlier draft where I actually did need to version digits separately...

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Review by jfgrimm — APPROVED** [review]

**Comment by jfgrimm** [comment]
Going in, thanks @boegel!

**Pull request merged**